service/service.go

```diff
@@ -46,13 +44,16 @@ func New(conf *config.Config, oracleAccount *panacea.OracleAccount) (*Service, e
 		return nil, fmt.Errorf("failed to init subscriber: %w", err)
 	}
 
+	txBuilder := panacea.NewTxBuilder(grpcClient)
+
 	return &Service{
 		conf:          conf,
 		oracleAccount: oracleAccount,
-		oraclePrivKey: oraclePrivKey,
+		oraclePrivKey: oraclePrivKeyBz,
```

Comment:
Do you have any specific reason to use []byte of oracle private key, not `*btcec.PrivKey`?

Reply:
I think we should also use `*btcec.PrivKey`.

Reply:
It's because when make registration vote, the type of private key in `types.OracleRegistration`.
I'll check the method that convert type `*btcec.PrivKey` to `[]byte`.

Reply:
You can convert to `[]byte` using `btcec.Privkey.Serialize()`.

Reply:
Thanks! I'll use `Serialize()` method.